src/main/java/seedu/address/logic/commands/DeletePersonCommand.java

```diff
@@ -24,6 +24,7 @@
             + "Example: " + COMMAND_WORD + " 1";
 
     public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
+    public static final String MESSAGE_DEBT_NOT_PAID = "Unable to delete, current person debt not cleared";
```

Comment:
I would recommend that you change this to "Unable to delete %1$s due to outstanding debts" and pass in the person's name when you use MESSAGE_DEBT_NOT_PAID